MAINTAINERS.yml

```diff
@@ -1967,6 +1967,23 @@ Retention:
   labels:
     - "area: Retention"
 
+Sensor Subsystem:
+  status: maintained
+  maintainers:
```

Comment:
Please include myself as a maintainer. We should have 1 Intel and 1 Google maintainer on this subsystem.

Reply:
add you with pleasure :)